### PR TITLE
mulle: Update RTT config to match other Kinetis boards

### PR DESCRIPTION
**Pull request by gebart**
The ISR config was broken

### CODE REVIEW

**Comment by gebart** [comment]
ping @kYc0o 
Could you take a look at this trivial fix?
The ISR name in periph_conf.h for mulle should be the same as the names in frdm-k64f, frdm-k22f and pba-d-01-kw2x

File: boards/mulle/include/periph_conf.h [hunk 1]
```diff
@@ -363,9 +363,9 @@ static const spi_conf_t spi_config[] = {
  * @{
  */
 #define RTT_NUMOF           (1U)
-#define RTT_IRQ_PRIO        CPU_DEFAULT_IRQ_PRIO
 #define RTT_IRQ             RTC_IRQn
-#define RTT_ISR             isr_rtc_alarm
+#define RTT_IRQ_PRIO        10
```
Comment (kYc0o) [inline boards/mulle/include/periph_conf.h]: Don't we have a macro for this level (or for levels in general)?

Comment (gebart) [reply boards/mulle/include/periph_conf.h]: There's a default IRQ prio Maceo which sets the priority to 1, but I copied this from the other kinetis boards to match. I figured any refactoring of the IRQ priorities should update all the boards in bulk.

Comment (kYc0o) [reply boards/mulle/include/periph_conf.h]: Ok, got it.

**Pull request merged**